arch/arc/core/thread.c

```diff
@@ -231,7 +231,6 @@ int arch_float_disable(struct k_thread *thread)
 	return 0;
 }
 
-
```

Comment:
you could discard this white space diff :)

Reply:
Oops... fixed.